From 93ec73dacd9e6f56442cb6fe9af2e26730cd5fc0 Mon Sep 17 00:00:00 2001
From: Joe Slater <jslater@windriver.com>
Date: Tue, 7 Mar 2017 14:53:19 -0800
Subject: [PATCH] e2fsprogs: expand @mkdir_p@

Add AC_SUBST to configure.ac.  @mkdir_p@ is currently
not expanded so no locale data is written into usr/share/locale.

Upstream-Status: Pending

Signed-off-by: Joe Slater <jslater@windriver.com>

---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 5e837c9..3dad32b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -855,6 +855,8 @@ AC_SUBST(PACKAGE)
 AC_SUBST(VERSION)
 
 AM_GNU_GETTEXT
+dnl @MKDIR_P@ is expanded in AM_GNU_GETTEXT
+AC_SUBST([mkdir_p],['$(MKDIR_P)'])
 dnl
 dnl End of configuration options
 dnl
